modules/nw-ipfailover-configuration.adoc

```diff
@@ -122,7 +122,7 @@ spec:
         node-role.kubernetes.io/worker: ""
       containers:
       - name: openshift-ipfailover
-        image: quay.io/openshift/origin-keepalived-ipfailover
+        image: registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9:v{product-version}
```

Comment:
@candita I am not sure did you meant 'registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9:v4.17`? or something like this?
am getting the sha version image from the registry like the below
`registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9@sha256:cd90dbc3bcd557303aca3ac19b9a22d0e46519cc5ff8736e5febd72768378927`

Reply:
@melvinjoseph86 they will need to pull the image with the version tag that matches their OpenShift version.  We can't document the sha hash because it changes over time.  This works for me:

```
$ docker pull registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9:v4.17
v4.17: Pulling from openshift4/ose-keepalived-ipfailover-rhel9
25c75c34b2e2: Already exists 
dd419db309f8: Pull complete 
96a64da1f690: Pull complete 
896e73f7840e: Pull complete 
Digest: sha256:85e1eba0166dad76ef68f59aa51b770a0d525a253b684b9ef1bdab6a1bfa4558
Status: Downloaded newer image for registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9:v4.17
registry.redhat.io/openshift4/ose-keepalived-ipfailover-rhel9:v4.17
```